app/forms/TelephoneNumberFormProvider.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import play.api.data.Form
+
+class TelephoneNumberFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[String] =
+    Form(
+      "value" -> text("telephoneNumber.error.required")
+        .verifying(
+          firstError(
+            isNotEmpty("value", "telephoneNumber.error.required"),
```

Comment:
Is `isNotEmpty()` required? Does it do the same behaviour as `text("telephoneNumber.error.required") as in it's mandatory.

Reply:
I'm not too sure on this one without looking at the implementation of `text()`